tools/codegen/model.py

```diff
@@ -304,6 +304,10 @@ def __post_init__(self) -> None:
             # TODO: fixme
             if str(self.name) not in [
                     '_amp_non_finite_check_and_unscale_',
+                    '_foreach_add_.ScalarList',
+                    '_foreach_sub_.ScalarList',
+                    '_foreach_mul_.ScalarList',
+                    '_foreach_div_.ScalarList',
```

Comment:
@bdhirsh This is the list of special cases I was referring to. Need to figure out what to do about this.